pkg/descriptions.json

```
@@ -1042,6 +1042,15 @@
         "System.Buffers.Text.Utf8Formatter"
     ]
   },
+  {
+    "Name": "Microsoft.Bcl.AsyncInterfaces",
+    "Description": "Provides interfaces and helper types for consuming and producing async enumerables on .NET Standard 2.0. This package is not required when targeting .NET Standard 2.1 or .NET CoreApp 3.0 as they already have support for these types inbox.",
```

Aligning with the text from `Microsoft.Bcl.HashCode`

> Provides the HashCode type for .NET Standard 2.0. This package is not required starting with .NET Standard 2.1 and .NET Core 3.0.

I'd go with:

```suggestion
    "Description": "Provides the IAsyncEnumerable<T> and IAsyncDisposable interfaces and helper types for .NET Standard 2.0. This package is not required starting with .NET Standard 2.1 and .NET Core 3.0",
```

/cc @mairaw